### PR TITLE
Product photo

### PR DESCRIPTION
**Pull request by zuzanna81**
## PT Link: https://www.pivotaltracker.com/story/show/154132379

### CODE REVIEW

**Review by lcegit — COMMENTED** [review]
Need to fix asset pipeline or add an image col to the product table.

**Comment by lcegit** [comment]
@aljoh @oliverochman do you know why our test is failing here?

**Review by NegativeKarma — COMMENTED** [review]
LGTM

File: features/restaurant_page.feature [hunk 1]
```diff
@@ -5,15 +5,15 @@ Feature: User can see restaurant's details on the restaurant show page
...
+      | Borat's Palace    | Östra Hmangatan 1 | Göteborg | The best borat you can get      |
 
   @googlemap
-  Scenario:
-    When I visit the landing page
-    And I click on "Visit" by "Thomas Kebabrulle"
-    Then I should be on the "Thomas Kebabrulle" show page
-    Then I would like to see "Thomas Kebabrulle"
-    And I would like to see "The best kebabrulle you can get"
+    Scenario:
```
Comment (diraulo) [inline features/restaurant_page.feature]: No need to indent the `Scenario` block after the `@googlemap` tag

The following is okay:
```gherkin
@googlemap
Scenario:
  When ...
```

**Review by diraulo — CHANGES_REQUESTED** [review]
Please fix the indentation in the feature files so we can get a clean code difference to review

**Comment by tochman** [comment]
What is the status of this one? Should we review this? Is anybody working on this?

**Pull request closed without merging**